autometa/common/kmers.py

```diff
@@ -70,14 +96,14 @@ def init_kmers(kmer_size=5):
             for char in dna_letters:
                 new_list.append(current_seq + char)
         all_kmers = new_list
-    # Now we trim k-mers and put them in the dictionary
```

Comment:
Might be useful to keep this comment here, minus the question, of course, and perhaps more clear.